src/main/java/org/example/util/PasswordEncoder.java

```diff
@@ -0,0 +1,14 @@
+package org.example.util;
+
+import org.springframework.security.crypto.argon2.Argon2PasswordEncoder;
+
+public final class PasswordEncoder {
+    private PasswordEncoder() { }
+
+    private static final Argon2PasswordEncoder ARGON_2_PASSWORD_ENCODER
+            = new Argon2PasswordEncoder(16, 32, 1, 60000, 10);
```

Comment:
Did checkstyle not complain about magic numbers here?

Reply:
It didn't complain. When I tried to change the numbers eg. 'private final int saltLength = 16;' it wouldn't accept it.